mods/xpanes/init.lua

```diff
@@ -54,10 +54,10 @@ local function update_pane(pos)
 	if count == 0 then
 		swap(pos, node, name .. "_flat", any)
 	elseif count == 1 then
-		swap(pos, node, name .. "_flat", (any + 1) % 4)
+		swap(pos, node, name .. "_flat", (any + 1) % 2)
```

Comment:
I'm very suspicious about modulo 2, as modulo 4 is to make rotations 0-3 cyclic.
Corner panes do not have 180 degree rotational symmetrey.
I also cannot see any benefit to using modulo 2 even if it is visually identical behaviour, the rotation value seems wrong.

Reply:
Ok, I'll change it back